updated info regarding reference impl and TELs (#140)
trustoverip/tswg-did-method-webs-specification: spec/informative_references.md

### Informative References

[[def: did:webs Reference Implementation]]

> ~ See the [python reference implementation of did:webs](https://github.com/hyperledger-labs/did-webs-resolver) that was developed as this specification was being created. Especially useful is the [GETTING STARTED guide](https://github.com/hyperledger-labs/did-webs-resolver/blob/main/GETTING_STARTED.md) which many of the examples were copied from.

Reviewer: ```suggestion
~ See the [python reference implementation of did:webs](https://github.com/hyperledger-labs/did-webs-resolver).   The [GETTING STARTED guide](https://github.com/hyperledger-labs/did-webs-resolver/blob/main/GETTING_STARTED.md) was created for new users and is the source that many of the examples in this specification were copied from.
```
